litellm/caching.py

```diff
@@ -123,7 +123,7 @@ def set_cache(self, key, value, **kwargs):
     async def async_set_cache(self, key, value, **kwargs):
         self.set_cache(key=key, value=value, **kwargs)
 
-    async def async_set_cache_pipeline(self, cache_list, ttl=None):
+    async def async_set_cache_pipeline(self, cache_list, ttl=None, **kwargs):
```

Comment:
why is this PR changing caching.py?  @superpoussin22

Reply:
@krrishdholakia  the commit for caching is not done by me if you look at the commits in the PR that's strange
and I see the changes in .docker ignore so better closing the PR